Dockerfile

```
@@ -1,4 +1,4 @@
-FROM golang:1.24-alpine3.22 as builder
+FROM golang:1.25-alpine3.22 as builder
 WORKDIR /app
 COPY . .
 RUN 
```

Hadolint (2.14.0)

[error] expecting the shell arguments

(DL1000)